cycledash/genotypes.py

```diff
@@ -397,14 +420,6 @@ def _header_spec(vcf_header_text, extant_cols):
                 'Names of genes that overlap with this variant\'s '
                 'starting position, derived from Ensembl Release 75.'))
 
-    # Remove empty supercolumns
```

Comment:
No, actually not -- this is just an artifact of the diff. These lines are still present a little bit above, but somehow this diff view thinks that they were deleted and added back.